web/tailwind.config.js

```diff
@@ -8,9 +8,43 @@ export default {
   theme: {
     extend: {
       fontFamily: {
-        sans: ['Montserrat'],
-        body: ['Open Sans'],
+        sans: ['Work Sans']
       },
+      fontSize: {
+        xlg: "37.46px", /*Welcome Back...*/
+        lg: "17.03px", /*Glad to see you're back...*/
+        md: "12.1px", /*Launch Night*/
+        sml: "7.26px", /*Monday 8th April...*/
+        xs: "4.98px" /*club Event*/
+      },
+      letterSpacing: {
+        
+      },
+      fontColor: {
+        primary: "#FFFFFF"
+      },
+      lineHeight: {
+        xlg: "43.94px", /*Welcome Back...*/
+        lg: "19.97px", /*Glad to see you're back...*/
+        md: "14.2px", /*Launch Night*/
+        sml: "8.52px", /*Monday 8th April...*/
+        xs: "5.84px" /*club Event*/
+      },
+      colors: {
+        primary: {
+          background: "#FFFEF9", /*white background*/
+          blue: "#3B87DD" /*main page bold blue + dashboard buttons etc*/
```

Comment:
![image](https://github.com/UoaWDCC/volunteers/assets/119158272/7e3ac395-c4e7-4629-a378-b7612314e0d0)

im thinking theres no need to nest the colours, and keep the names general in case we want to change the primary and secondary colours in the future 

```
colours: {
         primary: "#3B87DD", /* blue */
        "primary-dark": "#255894", /* dark blue */
        "primary-light": "#8BB4F9", /* light blue */
        secondary: "#F2504B", /* red */
        "secondary-light": "#FE8686", /* light red */
        neutral: "#FFFEF9", /* basically white */
        "neutral-tan": "#FAF6EB", /* tan colour */
        black: "#33342E", /* lighter black */
        grey: "#818181" /* grey */ 
}
```

Reply:
thanks